api/envoy/api/v2/route/route.proto

```diff
@@ -475,36 +475,28 @@ message CorsPolicy {
   google.protobuf.BoolValue allow_credentials = 6;
 
   oneof enabled_specifier {
-    // Specifies if CORS is enabled. Defaults to true. Only effective on route.
+    // Specifies if the CORS filter is enabled. Defaults to true. Only effective on route.
     //
     // .. attention::
     //
     //   **This field is deprecated**. Set the
     //   :ref:`filter_enabled<envoy_api_field_route.CorsPolicy.filter_enabled>` field instead.
     google.protobuf.BoolValue enabled = 7 [deprecated = true];
 
-    // Specifies if CORS is enabled.
+    // Specifies the % of requests for which the CORS filter is enabled.
```

Comment:
Sorry I didn't catch this before, but I think these docs should still say that the default if not specified is 100%? Same elsewhere?

Reply:
👍